README.md

```diff
@@ -6,8 +6,22 @@ For more details visit [the project page](https://wiki.openmrs.org/display/docs/
 
 ## Requirements
  * Maven 3.x
+    
```

Comment:
It's probably better to do several branches off of master rather than branching off as-yet unmerged PRs.